client/src/app/admin/panel/tickets/admin-panel-my-tickets.js

```diff
@@ -39,17 +44,34 @@ class AdminPanelMyTickets extends React.Component {
         );
     }
 
+    filterOpenedTickets(tickets){
```

Comment:
Filtering should be done in backend because pagination will broke (it will show less items than retrieved) and part of the purpose of this task it to retrieve less items to save load time.